packages/common/aws.js

```diff
@@ -558,20 +557,8 @@ exports.S3ListObjectsV2Queue = S3ListObjectsV2Queue;
 
 exports.checksumS3Objects = (algorithm, bucket, key, options = {}) => {
   const param = { Bucket: bucket, Key: key };
-
-  if (algorithm.toLowerCase() === 'cksum') {
-    return getFileChecksumFromStream(
-      exports.s3().getObject(param).createReadStream()
-    );
-  }
-
-  return new Promise((resolve, reject) => {
-    const hash = crypto.createHash(algorithm, options);
-    const fileStream = exports.s3().getObject(param).createReadStream();
-    fileStream.on('error', reject);
-    fileStream.on('data', (chunk) => hash.update(chunk));
-    fileStream.on('end', () => resolve(hash.digest('hex')));
-  });
+  const fileStream = exports.s3().getObject(param).createReadStream();
```

Comment:
Really nitpicky, but I kinda wish we could avoid having the `checksum` package dependency here just to support this function. But the only clean way to do that would be to deprecate this function and treat an S3 stream the same as any other input stream to `generateChecksumFromStream`, which would require refactoring any use of `checksumS3Objects`.

No action necessary, just thinking out loud.

Reply:
Hmm.. My personal priority is to make small focused packages like `checksum` to plug into `common` as needed, rather than have `common` be plugged into everything else. I don't disagree, though, but I also don't know of a clean way to do this.

Reply:
👍 for small, focused packages